frontend/src/App.css

```diff
@@ -187,6 +186,71 @@ body {
 }
 
 .footer button:active {
-    filter: brightness(150%);
-    box-shadow: 0 0 5px var(--persian-pink);
-}
\ No newline at end of file
+  filter: brightness(150%);
+  box-shadow: 0 0 5px var(--persian-pink);
+}
+/*Card*/
+.cards {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-gap: 20px;
+  margin: 1em auto;
+}
+
+.card {
+  display: grid;
+  padding: 5px;
+  border: 2px solid #999;
+  border-radius: 3px;
+  background-color: #fff;
+  width: 300px;
+}
+
+.card img {
+  object-fit: cover;
+  width: 100%;
+}
+
+.card h2 {
+  margin: 0;
+  padding: 0.5rem;
+}
+
+.card .content {
+  padding: 0.5rem;
+}
+
+.card footer {
+  background-color: rgb(3, 60, 9);
+  color: #fff;
+  height: 20px;
+  width: 300px;
+
+  text-align: center;
+}
+.grid-container {
+  display: grid;
+  grid-template-columns: auto auto auto;
+
+  padding: 4px;
+}
+
+.grid-container {
+  list-style: none;
+  margin: 2em;
+  display: grid;
+  gap: 20px;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+}
+.grid-item {
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  font-size: 15px;
+  text-align: center;
+}
+.pageHeader {
+  font-size: 35px;
+  color: brown;
```

Comment:
Brown isn't on the color palette that's been defined at the first row of the CSS file.